code/modules/surgery/implant.dm

```diff
@@ -224,6 +224,24 @@
 		affected.hidden.update_icon()
 		affected.hidden = null
 
+	else if (((M_CLUMSY in user.mutations) || user.reagents.has_reagent(INCENSE_BANANA) || user.reagents.has_reagent(HONKSERUM)) && prob(20)) //I HATE DATUMS! I HATE DATUMS!
+		user.visible_message("<span class='notice'>[user] takes something out of incision on [target]'s [affected.display_name] with \the [tool].</span>", \
+		"<span class='notice'>You take something out of incision on [target]'s [affected.display_name]s with \the [tool].</span>" )
+		var/obj/clowndigobj = pick(/obj/item/weapon/bikehorn/rubberducky, /obj/item/weapon/reagent_containers/food/snacks/pie, /obj/item/toy/singlecard, /obj/item/toy/waterflower)
+		clowndigobj = new clowndigobj(user.loc)
+		if (istype(clowndigobj, /obj/item/toy/singlecard))
+			var/obj/item/toy/singlecard/O = clowndigobj
+			O.cardname = "Joker"
```

Comment:
uhhh will this even work as is? shouldn't it be "Black Joker" or "Red Joker"?

Reply:
this was before slimes told me he made TWO OF THEM. calm down

Reply:
duh, decks of cards have two jokers